prboom2/src/m_menu.c

```diff
@@ -1189,81 +1189,6 @@ void M_Options(int choice)
   M_SetupNextMenu(&OptionsDef);
 }
 
-/////////////////////////////
-//
-// M_QuitDOOM
-//
```

Comment:
Why did you move this code?

Reply:
Beside moving this code, I've added the two lines:
```
setup_active = false;
currentMenu = NULL;
```
This is to avoid the overlay getting applied to the quit message when sending the SDL_Quit response while the overlay is open. The `M_QuitDOOM` code needed to be moved down below the `setup_active` declaration to change the variable status.

Reply:
It's easier to move the `setup_active` declaration to the beginning of the file. If you move a big chunk of code, we can't see in `diff` what was changed in it.

Reply:
Decided to move back `M_QuitDOOM` and move the menu active declarations to the top of the file. I'd rather not separate them, as the comments already have them grouped together.